frontend/src/lib/components/FlagSelector.tsx

```diff
@@ -10,9 +10,10 @@ interface FlagSelectorProps {
     value: number | undefined
     onChange: (id: number, key: string) => void
     readOnly?: boolean
+    readOnlyReason?: string
 }
 
-export function FlagSelector({ value, onChange, readOnly }: FlagSelectorProps): JSX.Element {
+export function FlagSelector({ value, onChange, readOnly, readOnlyReason }: FlagSelectorProps): JSX.Element {
```

Comment:
I'd keep this as `disabledReason` to match the lemon ui usage... signals it's the same thing